examples/Microsoft.Spark.CSharp.Examples/Sql/Datasource.cs

```diff
@@ -0,0 +1,202 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Policy;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.Sql
+{
+    /// <summary>
+    ///     A simple example demonstrating Spark SQL data sources
+    ///     A copy of spark/examples/src/main/python/sql/datasource.py
+    ///
+    ///     To run the jdbc example you need a valid jdb reference and driver, as an example to connect to postgres:
+    ///         - download the progress jdbc driver (https://jdbc.postgresql.org/download.html) 
+    ///         - add the jar to the jars you pass to spark-submit
+    ///         - as well as the path to the SPARK_HOME resources, also include the database:
+    ///             - jdbc url
+    ///             - username
+    ///             - password
+    ///             - the name of a table to read from
+    ///             - the name of a table to write data back to
+    ///
+    ///     spark-submit --class org.apache.spark.deploy.DotnetRunner `
```

Comment:
To be consistent with other examples, can we remove spark-submit command? (same below in the code)